### ISSUE TITLE
The durability totem in end goblin traders is still unavailable

### ISSUE THREAD

**Issue by Q2297045667** [Affehund/VoidTotem#9]
item ID:
endgoblintraders:durability_totem

MOD link:
https://www.curseforge.com/minecraft/mc-mods/end-goblin-traders-fabric

Minecraft 1.18 
end goblin traders 1.4.3 Hotfix
I think the big probability is the problem of void totem

![图片](https://user-images.githubusercontent.com/42416195/155847914-df99a348-f961-4ca5-b0d7-4fa61f1fb8cc.png)

I modified charm.json file, but it's useless
![图片](https://user-images.githubusercontent.com/42416195/155848087-0e080df2-3e63-41e9-88df-39564a0ebde6.png)

**Comment by Affehund** [comment]
So you mean like being able to use the end goblin totem to prevent death when falling in the void? 
That’s currently not possible and wasn’t part of the end goblin traders mod fix, which was about a mod conflict causing the void totem not to work at all. 
That’s something I would have to implement within the void totem mod. 
i thought your problem has been solved, but it was probably a misunderstanding.

**Comment by Q2297045667** [comment]
I'm sorry I didn't make it clear. The problem led to this misunderstanding

**Comment by Q2297045667** [comment]
I mean, can you support the durability totem in end goblin traders

**Comment by Affehund** [comment]
Yes, I can make it possible to be able to use the end goblin totem to prevent void death.

**Comment by Jab125** [comment]
this issue is invalid (the issue is that end goblin trader totems can't be used in trinkets slots) @Affehund

**Comment by Jab125** [comment]
> this issue is invalid (the issue is that end goblin trader totems can't be used in trinkets slots) @Affehund

which will be fixed when it is migrated to ThonkUtil's Totem API

**Comment by Affehund** [comment]
I will add an item tag where you can add additional void totems, e.g. the durability totem of end goblin traders, in the next version of the mod.
When adding an item to this tag you will be able to use that item as a void totem.

**Comment by Affehund** [comment]
I just released v1.2.0 (1.3.0 for 1.18.2) and with this version you can easily add additional totems via datapack.
Example for adding the totem of undying to additional_totems.json located in data/voidtotem/tags/item.
```json
{
  "replace": false,
  "values": [
    "minecraft:totem_of_undying"
  ]
}
```

**Comment by Q2297045667** [comment]
That's great. Thank you very much
